src/networking.rs

```diff
@@ -14,14 +17,120 @@ pub const RESP_TIMEOUT: Duration = Duration::from_secs(10);
 pub const ATTESTATION_PROPAGATION_SLOT_RANGE: usize = 32;
 pub const MAXIMUM_GOSSIP_CLOCK_DISPARITY: Duration = Duration::from_millis(500);
 
-pub use multiaddr::Multiaddr;
+/// Public keys with byte-lengths smaller than `MAX_INLINE_KEY_LENGTH` will be
```

Comment:
let's add a comment here that this was cribbed from `rust-libp2p` and that we should revisit the implementation

Reply:
done!